ext/datasource/consul/consul.go

```diff
@@ -0,0 +1,140 @@
+package consul
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/alibaba/sentinel-golang/ext/datasource"
+	"github.com/alibaba/sentinel-golang/logging"
+	"github.com/alibaba/sentinel-golang/util"
+	"github.com/hashicorp/consul/api"
+)
+
+type consulDataSource struct {
+	datasource.Base
+	*options
```

Comment:
It seems this field is useless for datasource, 
Deleting this field makes more sense.